File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-change-formatting-in-worksheet-rows-containing-selected-cells_8.cs
        // Keep track of the previously bolded row.
        static int previousRow = 0;

        private void BoldCurrentRow(Excel.Worksheet ws)
        {
            // Work with the current active cell.
            Excel.Range currentCell = this.Application.ActiveCell;

            // Bold the current row.
            currentCell.EntireRow.Font.Bold = true; 

            // If a pass has been done previously, make the old row not bold.
            // Make sure previousRow is not 0 (otherwise this is your first pass through).
            if (previousRow != 0)
            
                // Make sure the current row is not the same as the previous row.
                if (currentCell.Row != previousRow)
                {
                    Excel.Range rng = (Excel.Range)ws.Rows[previousRow];
                    rng.EntireRow.Font.Bold = false;
                }

            // Store the new row number for the next pass.
            previousRow = currentCell.Row;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-change-formatting-in-worksheet-rows-containing-selected-cells_8.vb
    Private Sub BoldCurrentRow(ByVal ws As Excel.Worksheet)

        ' Keep track of the previously bolded row.
        Static previousRow As Integer = 0

        ' Work with the current active cell.
        Dim currentCell As Excel.Range = Me.Application.ActiveCell

        ' Bold the current row.
        currentCell.EntireRow.Font.Bold = True

        ' If a pass has been done previously, make the old row not bold.
        ' Make sure previousRow is not 0 (otherwise this is your first pass through).
        If previousRow <> 0 Then

            ' Make sure the current row is not the same as the previous row.
            If currentCell.Row <> previousRow Then

                Dim rng As Excel.Range = DirectCast(ws.Rows(previousRow), Excel.Range)
                rng.EntireRow.Font.Bold = False
            End If
        End If

        ' Store the new row number for the next pass.
        previousRow = currentCell.Row
    End Sub